README.md

```diff
@@ -7,14 +7,14 @@ cd android-dpp
 /gradlew assemble
 ```
 - After building, it will be available on the local Maven repository.
-- To use it with gradle, add `mavenLocal()` to the `repositories` list in your `build.gradle` file and add `org.dashevo:dpp:0.19-SNAPSHOT` as a dependency. 
+- To use it with gradle, add `mavenLocal()` to the `repositories` list in your `build.gradle` file and add `org.dashevo:dpp:0.20-SNAPSHOT` as a dependency. 
```

Comment:
Is this the `org.dashevo` reference on this line supposed to change to `org.dashj.platform` also?

Reply:
good catch, that should have changed with 0.19.